crates/bevy_core/src/time/timer.rs

```diff
@@ -202,6 +202,10 @@ impl Timer {
     /// ```
     pub fn tick(&mut self, delta: Duration) -> &Self {
         if self.paused() {
+            self.times_finished = 0;
```

Comment:
This seems questionable. Presumably times_finished should be usable to get a sense of the total elapsed time.

Why did you need to make this change?

Reply:
Yeah, I was confused about times_finished as well. It's probably not what you think it is. It's not the amount of times a repeating timer has finished in total, but how many times it's finished this tick. I only realized that after reading it's doc comment. Maybe it should be renamed to avoid confusion, and we could probably add a times_finished that actually does what most people would expect it to.

Reply:
> Yeah, I was confused about times_finished as well. It's probably not what you think it is. It's not the amount of times a repeating timer has finished in total, but how many times it's finished within the last tick(i.e. almost always 0 or 1). I only realized that after reading it's doc comment. Maybe it should be renamed to avoid confusion, and we could probably add a times_finished that actually does what most people would expect it to.

I think we should just rename this here: I don't think the functionality is useful enough to add what I had in mind.